include/uacpi/platform/compiler.h

```diff
@@ -11,7 +11,7 @@
 
 #define UACPI_ALIGN(x) __declspec(align(x))
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Does this actually matter for performance? This just changes whether it uses pragma vs attributes

Reply:
No it doesn't, I changed that one back.